index.js

```diff
@@ -0,0 +1,19 @@
+const express = require('express')
+const app = express()
+const port = process.env.PORT || 5000
+
+app.get('/api/greetings/:name', (req, res) => {
+    res.status(200).send(`<h1>Hello ${req.params.name}</h1>`)
+})
+
+app.get('/api/greetings', (req, res) => {
+    res.status(400).json('please provide name (ex. /api/greeting/someName)')
+})
```

Comment:
Let's create an additional structure to the project:
```
src/main.js
src/modules
->src/modules/greetings
-->src/modules/greetings/constants/index.js
-->src/modules/greetings/routes/index.js
-->src/modules/greetings/controllers/sayHi.js
```
Both of the routes should be combined by common API prefix "/api/greetings" in main.js.
Routing of the greeting module should describe each route and execute right controller like middleware.
Also, could you please move all the messages/templates to the constants file?